pylint/checkers/base.py

```diff
@@ -2593,6 +2594,41 @@ def _check_type_x_is_y(self, node, left, operator, right):
         self.add_message("unidiomatic-typecheck", node=node)
 
 
+class ThreadingChecker(_BasicChecker):
+    """Checks for threading module
+
+    - useless with lock - locking used in wrong way that has no effect (with threading.Lock():)
+    """
+
+    msgs = {
+        "W9990": (  # TODO
```

Comment:
```suggestion
        "W2101": (  # TODO
```

I checked and `21xx` is still "free".

Reply:
Thanks, updated